src/domain/registration/notEligible/NotEligible.tsx

```diff
@@ -13,9 +18,19 @@ const NotEligible: FunctionComponent = () => {
           <p className={styles.description}>
             {t('homePage.hero.descriptionText')}
           </p>
-          <p className={styles.notEligible}>
-            {t('registration.notEligible.text')}
-          </p>
+          <div className={styles.notEligible}>
+            <Icon
+              className={styles.notEligibleFace}
+              src={adultFaceIcon}
+              alt="Non eligible really sad face"
```

Comment:
If we are serious about accessibility, I suppose this should be "Sad face" (and even translated..)?